src/test/java/seedu/address/logic/commands/book/AddBookCommandIntegrationTest.java

```diff
@@ -10,7 +10,7 @@
 import seedu.address.model.UserPrefs;
 import seedu.address.model.book.Book;
 import seedu.address.testutil.BookBuilder;
-import seedu.address.testutil.TypicalAddressBook;
+import seedu.address.testutil.TypicalLibTask;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code AddBookCommand}.
```

Comment:
Look good to me, the changes reflect libtask correctly. No instances of addressBook in this file :100: